packages/cli/src/commands/discovery/init.ts

```diff
@@ -0,0 +1,30 @@
+import fs from 'node:fs';
+import degit from 'degit';
+import { Command } from '@oclif/core';
+import { confirm } from '@inquirer/prompts'
+
+export default class Init extends Command {
+  static description = "Creates a discovery folder based on the starter.store template.";
+
+  static examples = ["$ yarn faststore discovery init"];
+
+  async run() {
+    const discoveryFolderExists = fs.existsSync('discovery');
+
+    if (discoveryFolderExists) {
+      const confirmOverride = await confirm({ message: 'It looks like you already have a discovery folder in your store. Do you want to override it?' });
+
+      if (!confirmOverride) return this.log('🛑 Interrupted initializing discovery');
+    }
+
+    const discoveryEmitter = degit('vtex-sites/starter.store', {
+      force: true
+    });
+
+    this.log('Pulling starter.store template...');
+
+    discoveryEmitter.clone('discovery').then(() => {
+      this.log('Discovery created successfuly! You can find it at discovery/');
```

Comment:
```suggestion
      this.log('Discovery created successfully! You can find it at discovery/');
```

Reply:
question: did you mean to add this "/" at the end?

Reply:
@lariciamota I did! Coming to think of it, the intention was to make sure `discovery` is messaged as a "folder" to the developer. I figured that leaving it as `discovery` would add a bit of confusion to the sentence, but we could go even further and leave it as `"discovery/"` or something similar. I'm up to suggestions!

Reply:
^ This got changed on https://github.com/vtex/faststore/pull/2373/commits/8b9cade833cf4a5699eca3737a4a36a02d6646ce since it now includes the ability to pass a custom path. The wording changed a bit due to that.